pos_lot_selection/static/src/js/popup/ProductScreen.js

```diff
@@ -0,0 +1,46 @@
+/*
+    Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+*/
+odoo.define("pos_lot_selection.ProductScreen", function (require) {
+    "use strict";
+
+    const ProductScreen = require("point_of_sale.ProductScreen");
+    const Registries = require("point_of_sale.Registries");
+    const NumberBuffer = require("point_of_sale.NumberBuffer");
+
+    const PosLotSaleProductScreen = (ProductScreen) =>
+        class extends ProductScreen {
+            async _clickProduct(event) {
+                const product = event.detail;
+                if (product.tracking !== "none") {
```

Comment:
With this condition, we're not calling `_getAddProductOptions` which is an issue if for example you have a product tracked by lot that needs to be weighed with an electronic scale.

Reply:
I don't know if that is possible when the product and tracked and weighed, usually when it is tracked it is not divisible 
if so this need to be tasted with the electronic weights